libcxx/include/__algorithm/ranges_minmax.h

```diff
@@ -89,7 +89,7 @@ struct __fn {
     // vectorize the code.
     if constexpr (contiguous_range<_Range> && is_integral_v<_ValueT> &&
                   __is_cheap_to_copy<_ValueT> & __is_identity<_Proj>::value &&
-                  __desugars_to_v<__less_tag, _Comp, _ValueT, _ValueT>) {
+                  __desugars_to_v<__totally_ordered_less_tag, _Comp, _ValueT, _ValueT>) {
```

Comment:
Do you really need a total order here? Since you don't seem to need that, it might make sense to keep `__less_tag`.